namerd/storage/consul/src/main/scala/io/buoyant/namerd/storage/consul/ConsulDtabStore.scala

```diff
@@ -38,19 +43,20 @@ class ConsulDtabStore(
               case Return(result) =>
                 val namespaces = result.value.map(namespace).toSet
                 updates() = Activity.Ok(namespaces)
-                cycle(result.index)
+                cycle(result.index, backoffs)
               case Throw(e: NotFound) =>
                 updates() = Activity.Ok(Set.empty[Ns])
-                cycle(e.rsp.headerMap.get(Headers.Index))
+                cycle(e.rsp.headerMap.get(Headers.Index), backoffs)
               case Throw(e: Failure) if e.isFlagged(Failure.Interrupted) => Future.Done
               case Throw(e) =>
                 updates() = Activity.Failed(e)
                 log.error("consul ns list observation error %s", e)
-                cycle(None)
+                val sleep #:: backoffs0 = backoffs
```

Comment:
The convention is usually that `backoffs0` represents the prior backoffs and `backoffs1` represent the next backoffs.  Alternatively, `rest` or `tail` is sometimes used to denote the tail of a stream.  So this could be
`val sleep #:: backoffs1 = backoffs0` or `val sleep #:: rest = backoffs`.